spec/presenters/application_presenter_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationPresenter do
+  include ApplicationHelper
```

Comment:
So we don't have to include this (as it is really not the concern of this presenter) we can always use this way to instantiate the presenter:

`described_class.new(crime_application)`

Maybe cleaner?